docs/log-collector/README.md

```diff
@@ -105,6 +108,42 @@ Verify installation with `kubectl tvk-log-collector --help`
 
         kubectl tvk-log-collector --clustered --keep-source-folder --log-level error
 
+- To collect logs by providing gvk :
+
+        kubectl tvk-log-collector --clustered --gvk '[{"group": "<group>", "version": "<version>", "kind": "<kind>"}, {...}]'
```

Comment:
Providing JSON string via flags is not user-friendly.
Take input like - 
```
--gvk [Group1/Version1/Kind1, Group2/Version2/Kind2]
```
If found anything easier than this, use that.

Reply:
done